Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./Footer.css";
+
+const FooterContainer = () => {
```

Comment:
good job, I love the icons you brought in for the stretch and to make this look so good.